db/migrate/20230118102510_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
Please use `text` data type for `text` attribute in comments table as it's mentioned here in this [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).